src/ls/driver.ts

```diff
@@ -27,12 +27,18 @@ export default class TrinoDriver
       return this.connection;
     }
 
+    var server = this.credentials.server
+    if (this.credentials.port) {
+      server = server + ":" this.credentials.port
+    }
```

Comment:
Hi @elferherrera here, I think I would like to keep server unchanged. You can write it as `location:port` ... similar to what you do through the trino cli.